rocks.thorium.Mercury.yml

```diff
@@ -0,0 +1,107 @@
+id: rocks.thorium.Mercury
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    version: '23.08'
+    directory: lib/ffmpeg
+    add-ld-path: .
+cleanup-commands:
+  - mkdir -p $FLATPAK_DEST/lib/ffmpeg
+command: mercury-browser
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pcsc
+  - --socket=pulseaudio
+  - --socket=cups
+  - --persist=.mercury
+  - --filesystem=xdg-download:rw
+  - --filesystem=xdg-run/pipewire-0
+  - --device=dri
+  - --talk-name=org.freedesktop.FileManager1
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --talk-name=org.a11y.Bus
+  - --talk-name=org.gnome.SessionManager
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.gtk.vfs.*
+  - --talk-name=org.freedesktop.Notifications
+  - --env=MOZ_ENABLE_WAYLAND=1
+  - --env=MOZ_USE_XINPUT2=1
+  - --own-name=org.mpris.MediaPlayer2.firefox.*
+
+modules:
+  - shared-modules/dbus-glib/dbus-glib.json
+
+  - name: libnotify
+    buildsystem: meson
+    config-opts:
+      - -Dtests=false
+      - -Dintrospection=disabled
+      - -Dman=false
+      - -Dgtk_doc=false
+      - -Ddocbook_docs=disabled
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/libnotify/0.8/libnotify-0.8.3.tar.xz
+        sha256: ee8f3ef946156ad3406fdf45feedbdcd932dbd211ab4f16f75eba4f36fb2f6c0
+        x-checker-data:
+          type: antiya
+          project-id: 13149
+          url-template: https://download.gnome.org/sources/libnotify/$major.$minor/libnotify-$version.tar.xz
```

Comment:
Use the Firefox baseapp instead of adding dependencies here https://github.com/flathub/org.mozilla.firefox.BaseApp It includes other libraries needed too.

Reply:
Ok, I used [org.mozilla.firefox.BaseApp](https://github.com/flathub/org.mozilla.firefox.BaseApp)